src/main/pages/overview/infra-support.md

```diff
@@ -11,21 +11,21 @@ folder: overview
-- Kubernetes (soon)
+- Kubernetes
 
 Each of the infrastructures is providing its own specificities which are detailed in this page.
 
 ### Comparison Table
 
-|  Feature                 | **Docker**   | **OpenShift**                                                                              |
-| root access              | yes          | no (See: [Configuration](openshift-config.html#enable-ssh-and-sudo))                       |
-| https                    | no           | yes (See: [Configuration](openshift-config.html#https-mode))                               |
-| scalability              | no           | yes (See: [Configuration](openshift-config.html#scalability))                              |
-| priviliged containers    | yes          | no (configurable in [OpenShift](https://docs.openshift.com/container-platform/3.6/admin_guide/manage_scc.html#grant-access-to-the-privileged-scc))         |
-| health checks            | no           | yes                                                                                        |
-| persistent preview URLs  | no           | yes                                                                                        |
-| installers               | yes          | some installers may require [sudo access](openshift-config.html#enable-ssh-and-sudo)       |
-| file system permissions  | not limited  | limited to directories owned by root [group](openshift-config.html#filesystem-permissions) |
+|  Feature                 | **Docker**   | **OpenShift**                                                                              | **Kubernetes** |
+| root access              | yes          | no (See: [Configuration](openshift-config.html#enable-ssh-and-sudo))                       | yes |
+| https                    | no           | yes (See: [Configuration](openshift-config.html#https-mode))                               | yes |
+| scalability              | no           | yes (See: [Configuration](openshift-config.html#scalability))                              | yes |
+| priviliged containers    | yes          | no (configurable in [OpenShift](https://docs.openshift.com/container-platform/3.6/admin_guide/manage_scc.html#grant-access-to-the-privileged-scc))         | yes |
+| health checks            | no           | yes                                                                                        | yes |
+| persistent preview URLs  | no           | yes                                                                                        | yes |
+| installers               | yes          | some installers may require [sudo access](openshift-config.html#enable-ssh-and-sudo)       | yes |
```

Comment:
Again - change text for OpenShift to "yes, some installers may require..."

Reply:
Fixed